setup.ps1

```diff
@@ -0,0 +1,229 @@
+# Set the console color to Matrix theme and clear the console
+$host.UI.RawUI.BackgroundColor = "Black"
+$host.UI.RawUI.ForegroundColor = "Green"
+Clear-Host
+
+# Set the log file path and initialize variables
+$LogFilePath = Join-Path $env:TEMP "script_log.txt"
+$ProjectDir = "G:\Downloads\GitHub\freqtrade"
```

Comment:
this will not work - everyone will have a different path - so this MUST be something relative - or be gone completely.

if we expect users to update this - then it's no use, really - as it'll create a dirty state, and should we have to update this file, then the repo state will be "dirty" - causing subsequent `git pull`'s to fail.

Reply:
Haha, completely overlooked that. 😅